### PR TITLE
Remove handling of broken unittests

### PR DESCRIPTION
**Pull request by kiblik**
There are no broken unittests. It is not needed run these tests separately.

### CODE REVIEW

**Review by Maffooch — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Comment by mtesauro** [comment]
@kiblik Happy to merge this once the merge conflict is resolved.

**Pull request merged**
